openapi.yaml

```diff
@@ -3674,6 +3674,44 @@ paths:
       - lang: CLI
         source: >
           linode-cli databases mysql-backup-view 123 456
+    delete:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Backup Delete
+      operationId: deleteDatabaseMySQLInstanceBackup
+      servers:
+      - url: https://api.linode.com/v4
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-backup-delete
+      x-linode-grant: unrestricted only
+      description: |
+        Delete a single backup for an accessible Managed MySQL Database.
+
+        Requires `read_write` access to the Database.
+
+        The Database must not be provisioning to perform this command.
```

Comment:
The only statuses valid for this action are active, failed and degraded. Do we want to note that here?